ui/src/app/flags/Flag.tsx

```diff
@@ -32,7 +33,8 @@ import {
 import Rollouts from './rollouts/Rollouts';
 const tabs = [
   { name: 'Variants', to: '' },
-  { name: 'Rules', to: 'rules' }
+  { name: 'Rules', to: 'rules' },
+  { name: 'Analytics', to: 'analytics' }
```

Comment:
the analytics tab seems to be missing for Boolean flag types or maybe with no rollout rules?

![CleanShot 2024-02-05 at 09 37 33](https://github.com/flipt-io/flipt/assets/209477/a10d356d-cf9b-4d5b-a0e5-68c29ac1d0b0)